### DISCUSSION TITLE
hot reload the browser when change css/app.scss or js/app.js not work?

### DISCUSSION THREAD

**Discussion by zw963** [luckyframework/lucky#1912] [Q&A]
I don't know if this is a config issue, but, when change crystal source code, hot reload on the browser is working, but, not work when change CSS or JS,  assets was recompiled after changes happen, as the log  in following screenshot.

![image](https://github.com/user-attachments/assets/ef87ccf2-2e1a-42e6-bded-560ece1d3636)

but i have to refresh the browser to make CSS/JS take effect.

 I test on an new created lucky project, same, test on both Firefox and Chrome.

I saw a `--hot` options in `mix watch`.

```
 ╰─ $ yarn watch --help
yarn run v1.22.22
$ yarn run mix watch --help
$ /home/zw963/Crystal/crystal-china/one_in_ten_thousand/test123/node_modules/.bin/mix watch --help
Usage: mix watch [options]

Build and watch files for changes.

Options:
  --hot       Enable hot reloading. (default: false)
  --https     Enable https. (default: false)
  -h, --help  display help for command
Done in 0.17s.
```

But, set it not help.

Just curious, why we are not add the browser auto reload when change assets, Or, some things wrong on my side?

lucky version: 1.2.0

Thanks.

**Comment by jwoertink** [comment]
I don't have any apps that use laravel-mix v6. It's possible this is just a bug with the newer version. I use laravel-mix v4 and hot reload is working still.

**Comment by zw963** [comment]
> I don't have any apps that use laravel-mix v6.

I saw the laravel-mix version of lucky website update to 6.x.

https://github.com/luckyframework/website/blob/main/package.json#L21

Did I misunderstand something?

**Reply by jwoertink** [reply]
> Did I misunderstand something?

No, you did not misunderstand. I'm just saying that I don't know how to fix this issue you're having.
